fdi-x86/nls/SETUP.TR

```diff
@@ -1,36 +1,36 @@
 # Welcome Message
-WELCOME_DEF=/fLightGreen %1 /fCyan %2 /fGray kurulum program�na ho� geldiniz.
-WELCOME_ADV= /fLightGreen %1 /fCyan %2 /fGray /fLightRed geli�mi� /fGray kurulum program�na ho� geldiniz.
+WELCOME_DEF=/fLightGreen %1 /fCyan %2 /fGray "kurulum program�na ho� geldiniz."
+WELCOME_ADV= /fLightGreen %1 /fCyan %2 /fGray /fLightRed geli�mi� /fGray "kurulum program�na ho� geldiniz."

```

Comment:
You could also quote the `geli�mi�` _(github doesn't like DOS ascii)_ if you think it makes it easier to read the source string.

Reply:
Note to self -- If lazy me would just scroll down and comment on the UTF versions, the lack of DOS ASCII support in GitHub would not be a problem. :-)